src/wallet.ts

```diff
@@ -1,32 +1,47 @@
-export default {
-  "0x435933c8064b4ae76be665428e0307ef2ccfbd68": {
+type Account = {
```

Comment:
@claytonneal this will break the generate-accounts script. The script adds the accounts here. 

Maybe we can modify the script and remove it.

Reply:
ok will take a look, didnt know about that script - thanks

Reply:
Maybe we can just change the script to print the accounts. A dev can add them here

Reply:
i removed the script, and updated readme